library/swift/src/filters/Filter.swift

```diff
@@ -26,7 +26,7 @@ extension EnvoyHTTPFilter {
         case .continue(let headers):
           return [kEnvoyFilterHeadersStatusContinue, headers.headers]
         case .stopIteration:
-          return [kEnvoyFilterHeadersStatusStopIteration, envoyHeaders]
+          return [kEnvoyFilterHeadersStatusStopIteration, NSNull()]
```

Comment:
I think you can use `nil` and it will become `NSNull()` automagically when mapped to Objective-C.

Reply:
Alternatively it might still just be clearer to handle a different tuple length for `StopIteration`.

Reply:
the compiler complained about nil, because it is a 0 value rather than an object that conform to Any.

The rest of the code is pretty nice in its absence of branching so I felt it was good to leave this way?

Reply:
Does making the type `[Any?]` resolve this? Using `NSNull` explicitly feels a little bit against the grain.

Reply:
talked offline. Decided to leave NSNull to not leak branching into the static bridge functions.